Assets/Scripts/Game/LevitateMotor.cs

```diff
@@ -68,17 +68,25 @@ private void Update()
             if (GameManager.Instance.PlayerEntity.IsParalyzed)
                 return;
 
+            float inputX = 0f;
+            float inputY = 0f;
             // Forward/backwards
             if (InputManager.Instance.HasAction(InputManager.Actions.MoveForwards))
-                AddMovement(playerCamera.transform.forward);
-            else if (InputManager.Instance.HasAction(InputManager.Actions.MoveBackwards))
-                AddMovement(-playerCamera.transform.forward);
+                inputY++;
```

Comment:
Not sure about using ++ on float, to be honest. Minor issue though

Reply:
That's what was suggested by VS Code...

Reply:
Removed by the simplification suggestion by @jefetienne